media-fonts/caligula/caligula-1.0.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="A script font"
+DESCRIPTION="Caligula is a crude representation of what was originally a \
+calligraphic bitmap script font autotraced by Fontographer 3.2. There are no \
+kerning pairs in this font and it should probably be used only in sizes 24pt \
+or larger."
+HOMEPAGE="https://github.com/FontCollection/Caligula"
+COPYRIGHT="1994 or earler"
+LICENSE="Public Domain"
+REVISION="1"
+SOURCE_URI="https://github.com/FontCollection/Caligula/releases/download/initial/Caligula.tar.xz"
+CHECKSUM_SHA256="dde42b6f09b85e92c2723d695fce907f0a07a74dff35f6b6094571fa43ac125c"
+SOURCE_FILENAME="Caligula.tar.xz"
+SOURCE_DIR="Caligula"
+
+ARCHITECTURES="any"
+
+PROVIDES="
+	caligula = $portVersion
+	"
+
+BUILD_PREREQUIRES="
+	cmd:cp
+	cmd:mkdir
```

Comment:
Haven't checked a build, but I guess these 2 are provided by default and not needed?

Reply:
You are correct.  I just tried it without these and it worked.  Once again, copied from an existing font recipe.  Oh well.

Reply:
No problem, depending on the recipe you used as a base it could be an older one. :) Thank for checking :+1: